content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx

```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
 
 ## Baseline your production environment
 
-Assuming you have successfully transitioned to the new version of Prisma Migrate and initialized a new migration history in your repository, you now need to **baseline** any database that contains important data - such as production or staging.
+Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so it won't try to drop and recreate existing tables.
```

Comment:
Failing to baseline will not manifest in dropping and recreating tables, only failed migrations (the migration we would apply would try to create tables etc. that already exist).

Reply:
```suggestion
Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so the generated migration(s) won't fail when they will try to create tables that already exist.
```

Reply:
Better?